mscore/scoreview.cpp

```diff
@@ -3613,12 +3613,6 @@ void ScoreView::contextPopup(QContextMenuEvent* ev)
                   objectPopup(gp, e);
                   }
             }
-      else {
```

Comment:
FWIW, I've long thought the context menu for an empty spot in the score should contain items from the Style and Page Layout menu.

The option to add title etc is also worth keeping - that doesn't depend on a selection.  Similarly for appending measures & frames.